docs/development.md

```diff
@@ -171,11 +171,18 @@ Finally, update docs.
 
 ### External Tests
 
-If there are any external kola tests that were using the now stabilized experimental spec that are not part of the Ignition repo (e.x. tests in the [fedora-coreos-config](https://github.com/coreos/fedora-coreos-config/tree/testing-devel/tests/kola) repo) then CI will fail for the spec stabilization PR.
-
-- Add any tests using the experimental spec to the [fedora-coreos-config denylist](https://github.com/coreos/fedora-coreos-config/blob/testing-devel/kola-denylist.yaml) and re-run CI
-- Merge the spec stabilization PR
-- Perform a new Ignition release
-- Bump the Ignition packaging
-- Vendor the new Ignition release into [coreos-assembler](https://github.com/coreos/coreos-assembler)
-- Update the lockfile on the testing-devel branch in [fedora-coreos-config](https://github.com/coreos/fedora-coreos-config) to use the new Ignition release & the external test to use the new stabilized spec version, removing it from the denylist.
+If there are any external kola tests that were using the now stabilized experimental spec that are not part of the Ignition repo (e.x. tests in the [fedora-coreos-config](https://github.com/coreos/fedora-coreos-config/tree/testing-devel/tests/kola) repo) then CI will fail for the spec stabilization PR. You can temporarily add the following to `.cci.jenkinsfile` before the `fcosKola` step (replacing `3.3.0` with the config version being stabilized):
+
+```groovy
+// temporarily replace 3.3.0-experimental tests on-the-fly with 3.3.0
+shwrap("""
```

Comment:
Rather than putting this here, should we just leave it in `.cci.jenkinsfile` but commented out?